config/zones/CA-NU.yaml

```diff
@@ -4,4 +4,4 @@ bounding_box:
   - - -60.585479295999846
     - 83.61652252800013
 flag_file_name: ca.png
-timezone: null
+timezone: America/Iqaluit
```

Comment:
We plan on using these to improve our solar estimation models by introducing more data about daytime vs nightime. 

This data is provided in local timezones (as far as I know) so we'll only use these to normalize the time to UTC for now.

But yeah since we don't have any other data for the zone it might be better to set it as Null.

Reply:
I think Nunavut and eastern Russian timezones are going to be extreme edge cases for solar estimation models :sweat_smile: Nunavut's population centre longitudes span from 68 degrees W to 105 degrees W, plus there's high seasonal variation.

If you want to have a timezone here, it would perhaps be most accurate to use `America/Rankin_Inlet` since it's roughly in the middle of the territory? And add a comment like for Kazakhstan.